src/dataIntegration/metabotools/prepIntegrationQuant.m

```diff
@@ -73,8 +73,6 @@ function prepIntegrationQuant(model,metData,exchanges,samples,test_max,test_min,
      if ~isempty(G)
       Secretion_not_possible(l,1)= FBA_all_secreted_names{i};
       l=l+1;
-     else
-         Secretion_not_possible = {};
```

Comment:
Its in a loop, and 
 else
-         Secretion_not_possible = {};
kills the whole sense of it .... 


We can put this  Secretion_not_possible = {}; on top of the loop. 

 Secretion_not_possible = {};
 l=1;
 for i=1:length(FBA_all_secreted)
     G = FBA_all_secreted_names(find(FBA_all_secreted(i,1)==0));
     if ~isempty(G)
      Secretion_not_possible(l,1)= FBA_all_secreted_names{i};
      l=l+1;
          
     end
 end